app/models/user_annotation.rb

```diff
@@ -14,12 +14,39 @@ class UserAnnotation
   ###
 
   include Mongoid::Document
+  include Swagger::Blocks
+
   field :name, type: String
   field :values, type: Array
   field :queued_for_deletion, type: Boolean, default: false
   field :publishing, type: Boolean, default: false
   field :source_resolution, type: Integer
 
+  # Define parameters used in API POST endpoint to create user annotations
+  swagger_schema :UserAnnotation do
```

Comment:
Naming quibble - since this is not the actual `UserAnnotation` document structure as defined above, we should call this something different, like `UserAnnotationInput` to avoid confusion with the actual model.

Reply:
Good point, I had been waffling around on this.  Done in 296cd75.